docs/build/guides/freighter/connect-testnet.mdx

```diff
@@ -10,4 +10,4 @@ description: Create a Stellar account on Testnet with Freighter.
 
 3. Next, switch to Testnet. Testnet is available from the network dropdown.
 
-4. If your account does not exist on the selected network, Freighter will prompt you to fund it the account using Friendbot. Alternatively, you can do so in the Stellar [Laboratory](https://laboratory.stellar.org/#account-creator).
+4. If your account does not exist on the selected network, Freighter will prompt you to fund it the account using Friendbot. Alternatively, you can do so in the [Stellar Lab](https://lab.stellar.org/account/create?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;).
```

Comment:
We could omit the network settings to make the URL shorter. We'll get user's network setting from the local storage, if it exists, or use testnet by default.

```suggestion
4. If your account does not exist on the selected network, Freighter will prompt you to fund it the account using Friendbot. Alternatively, you can do so in the [Stellar Lab](https://lab.stellar.org/account/create).
```